js/product-page.js

```diff
@@ -1,124 +1,154 @@
+      data["drinks"][productId].name;
+
+    document.getElementsByClassName("product-page-name")[0].innerHTML =
+      data["drinks"][productId].category + " " + data["drinks"][productId].name;
+    document.getElementsByClassName("product-page-price")[0].innerHTML =
+      data["drinks"][productId].price.toFixed(2) + " uan.";
+
+    const inStock = data["drinks"][productId].inStock;
+    if (inStock == true) {
+      document.getElementsByClassName("product-page-stock")[0].innerHTML =
+        "on the shelf";
+      document.getElementsByClassName(
+        "product-page-stock-description"
+      )[0].innerHTML =
+        "The goods are in the store. You can pick it up now (until 21:00) or place a reservation for tomorrow.";
+      document.getElementsByClassName("product-page-stock-again")[0].innerHTML =
+        "In stock";
+    } else {
+      document.getElementsByClassName("product-page-stock")[0].innerHTML =
+        "on order";
+      document.getElementsByClassName(
+        "product-page-stock-description"
+      )[0].innerHTML =
+        "Make a reservation and the product will be available within 48 hours.";
+      document.getElementsByClassName("product-page-stock-again")[0].innerHTML =
+        "Preorder";
+    }
+    document.getElementsByClassName("product-page-characteritics-text")[0].innerHTML = data["drinks"][productId].description;
+
+    showcharacteristics(data);
+  };
+
+  const showcharacteristics = function (data) {
+    productCharacteristics = data["drinks"][productId].characteristics;
+    for (key in productCharacteristics) {
+      if (productCharacteristics.hasOwnProperty(key)) {
+        let characteristicsConteiner = document.getElementsByClassName("product-page-characteristics")[0];
+        if (productCharacteristics[key].link === true) {
+          let container = document.createElement("div");
+          container.classList.add("characteritics-container");
+          showCteristicName(productCharacteristics[key].name, container);
+
+          if (Array.isArray(productCharacteristics[key].value)) {
+            productCharacteristics[key].value.forEach(function (element) {
+              showCteristicValueBatton(element, container);
+              showCteristicValueSeparator(container);
+            })
+            characteristicsConteiner.appendChild(container);
+          } else {
+            showCteristicValueBatton(productCharacteristics[key].value, container);
+            characteristicsConteiner.appendChild(container);
+          }
+        }
+        else {
+          let container = document.createElement("div");
+          container.classList.add("characteritics-container");
+          showCteristicName(productCharacteristics[key].name, container);
+          showCteristicValue(productCharacteristics[key].value, container);
+          characteristicsConteiner.appendChild(container);
+        }
+      }
+    }
+  }
+  const showCteristicName = function (name, container) {
```

Comment:
characteristic? 
maybe it's better to replace all "characteristics" words with "property" - it's shorter and simplier